far2l/bootstrap/trash.sh

```diff
@@ -14,6 +14,12 @@ if [ -x ~/.config/far2l/trash.sh ]; then
 . ~/.config/far2l/trash.sh
 fi
 
+if command -v trash put "$1" 2>"$2"; then
```

Comment:
There is no need to pass arguments to `command -v trash`, they will be ignored anyway. Also let's ignore stdout and redirect stderr to stdout, for consistency with the blocks below.

```suggestion
if command -v trash >/dev/null 2>&1; then
```